packages/venia-ui/lib/components/Accordion/accordion.css

```diff
@@ -0,0 +1,11 @@
+.root {
+    border: 1px solid lightgray;
+    border-radius: 0.5em;
+}
+
+.root > * {
+    border-bottom: solid 1px lightgray;
+}
```

Comment:
This is valid CSS, but we don't use this approach in our components. Prefer putting these styles directly on some classnames in `section.css`.

If you really want to emphasize that `accordion.css` is the source of the styles, create some classnames in this file and have the classnames in `section.css` extend them with `composes`.

Reply:
Fixed in https://github.com/magento/pwa-studio/pull/2090/commits/507c44e2d3ff7678f3c5afd7a9a2a1bb4109e650.